misk-zookeeper/src/main/kotlin/misk/zookeeper/ZookeeperEnsembleModule.kt

```diff
@@ -0,0 +1,22 @@
+package misk.zookeeper
+
+import misk.clustering.zookeeper.ZookeeperConfig
+import misk.inject.KAbstractModule
+import misk.inject.keyOf
+import org.apache.curator.ensemble.EnsembleProvider
+import org.apache.curator.ensemble.fixed.FixedEnsembleProvider
+import javax.inject.Provider
+import kotlin.reflect.KClass
+
+class ZookeeperEnsembleModule(
```

Comment:
this should be called FixedEnsembleModule or something

Reply:
Yeah, I'll rename this to something better